accounts-tool/src/main.rs

```diff
@@ -1 +1,30 @@
-fn main() {}
+#![allow(clippy::integer_arithmetic)]
+use {
+    clap::{crate_description, crate_name, App, AppSettings},
+    log::*,
+    solana_measure::measure::Measure,
+    std::process::exit,
+};
+
+#[allow(clippy::cognitive_complexity)]
+fn main() {
+    solana_logger::setup_with_default("solana=info");
+
+    let mut measure_total_execution_time = Measure::start("accounts tool");
+    let matches = App::new(crate_name!())
+        .about(crate_description!())
+        .version(solana_version::version!())
+        .setting(AppSettings::InferSubcommands)
+        .setting(AppSettings::SubcommandRequiredElseHelp)
+        .setting(AppSettings::VersionlessSubcommands)
+        .get_matches();
+    match matches.subcommand() {
```

Comment:
One other touch that I think is nice is adding a line that spits out the name of the tool and version; see
https://github.com/solana-labs/solana/blob/db10073c3686406da457b274bc03923c0a44671c/ledger-tool/src/main.rs#L2235

This would go after `get_matches()` and before `match matches.subcommand()`